File: CppCompiler/forloop2_2.cpp
#include<iostream>
using namespace std;
bool isprime(int n)
{
int i,flag=0;
for(i=2;i<=n/2;i++)
{
if(n%i==0)
{
flag++;
return false;
break;
}
}
if(flag==0)
return true;
}




int main()
{
 double i,n,j,sum=0,k=2;
cout<<"enter no :-";
cin>>n;
for(i=1;i<=n;i++)
{
for(j=k;i<=n;j++)
{
if(isprime(j))
{
cout<<i<<"/"<<j;
if(i<n)
{
cout<<" + ";
}
sum=sum+i/j;
k=j+1;
break;
}
}
}
cout<<"sum of seriese="<<sum<<endl;
return 0;
}

File: JavaNIDE/Javaprgm/app/src/main/java/sample/Forloop22.java
package sample;
import java.util.*;
class Forloop22
 {
   
 public static void main (String[] ar)
 {
 int i,j,n;
 double sum1=0,sum=0,z,prod=1;
 System.out.println("enter no of term");
 Scanner m=new Scanner(System.in);
 n=m.nextInt();
 for(i=1;i<=n;i++)
 {
   
   for(j=1;j<=i;j++)
   {
    System.out.print(j+"^"+j);
    
      sum1=sum1+Math.pow(j,j);
    if(j<i)
    {
      System.out.print("+");
    }
    }
    
     System.out.print("/");

    for(j=1;j<=i;j++)
    {
     System.out.print(j);
     if(j<i)
     {
      System.out.print("*");
     }
     prod=prod*j;
     
    }
    
    if(i<n)
    {
      System.out.print(" + ");
    }
    sum=sum+(sum1/prod);
     sum1=0;
     prod=1;
   
 }
 System.out.print("sum of seriese ="+sum);
  
 }
  
}


